app/views/styles/component/_form_footer.scss

```diff
@@ -145,6 +145,18 @@
         &__main-controls {
             margin-bottom: 0;
             margin-top: 0;
+
+            .next-prompt {
+                border: none;
+                
+                input[type=checkbox] {
+                    float: none;
+                }
+
+                span {
+                    vertical-align: top;
+                }
+            }
         }
     }
 
```

Comment:
There is an issue with non-grid themes (eg. the OC theme):

<img width="1035" alt="Screen Shot 2020-11-16 at 3 39 50 PM" src="https://user-images.githubusercontent.com/627350/99316822-3cdc4600-2822-11eb-810f-675227fb21fe.png">

Reply:
In the Grid theme, would be good to tweak the hover behavior. Doesn't look so good.

<img width="1549" alt="Screen Shot 2020-11-16 at 3 12 54 PM" src="https://user-images.githubusercontent.com/627350/99316936-6bf2b780-2822-11eb-8e28-945ade3f5375.png">

At least override the yellow background (to transparent) I think

Reply:
style updated
using oc theme
![Screenshot from 2020-11-18 15-01-30](https://user-images.githubusercontent.com/13865076/99502975-60cd9200-29b0-11eb-87a9-f19db08a4144.png)
using grid theme
![Screenshot from 2020-11-18 15-05-55](https://user-images.githubusercontent.com/13865076/99503091-7f338d80-29b0-11eb-98e0-5fd2bffeaf0e.png)
is it okay to reduce the length of the background color when hovering?

Reply:
Yes, I think that's great.